tests/test_pytype.py

```diff
@@ -0,0 +1,15 @@
+import subprocess
+import sys
+import pytest
+
+
+@pytest.mark.skipif(
+    sys.version_info >= (3, 13), reason="Need python < 3.13 for pytype"
+)
+def test_pytype():
```

Comment:
I think ideally we would just call pytype directly from tox.ini, but I don't know whether it's possible to only run the test for Python < 3.13. Can you check?

Reply:
I researched around and tried a few things, but nothing seemed to work. I feel there might be a way, but it may not really be supported by tox's default behaviour. I suppose once pytype and msgspec support 3.13, we could switch to a direct call in tox?

Reply:
Thanks for trying! I agree that using pytest to call pytype for now is the best option.